src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -261,6 +261,9 @@
         - name: --fqdn-subdomain
           type: string
           short-summary: Prefix for FQDN that is created for private cluster with custom private dns zone scenario.
+        - name: --enable-public-fqdn
+          type: bool
+          short-summary: (Preview) Enable public fqdn feature for private cluster.
```

Comment:
There is no need to add label `(Preview)` in help, please add B in the registration of arguments. https://github.com/Azure/azure-cli-extensions/pull/3465/files#r646229907

Reply:
Sure. I'll still keep the `(Preview)` to keep consistency with others.